packages/dx-react-scheduler/src/plugins/day-view.jsx

```diff
@@ -27,15 +27,15 @@ export class DayView extends React.PureComponent {
     super(props);
 
     this.state = {
-      dateTableRef: null,
+      timeTableRef: null,
     };
 
-    this.dateTableRef = this.dateTableRef.bind(this);
+    this.timeTableRef = this.timeTableRef.bind(this);
 
     this.sidebarPlaceholder = () => <TemplatePlaceholder name="sidebar" />;
     this.dayScalePlaceholder = () => <TemplatePlaceholder name="navbar" />;
```

Comment:
Should we rename the `name` of the TemplatePlaceholder? And `sidebarPlaceholder` -> `timeScalePlaceholder` too?

Reply:
We didn't discuss `Template`, `Getter`, `Action` names yet. Well, I kept it as is.